qdrant-landing/content/documentation/embedding/gemini.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Gemini
+weight: 700
+---
+
+# Gemini
+
+Qdrant is compatible with Gemini Embedding Model API and its official Python SDK that might be installed as any other package:
+
+Gemini is a new family of Google PaLM models, released in December 2023. The new embedding models succeed the previous Gecko Embedding Model. 
+
+The new models allow passing an additional parameter to the API call: `task_type`. It determines the type of task you want to use the embeddings for.
+
+These are the task types that are supported by the Embedding Model API:
+
+- TASK_TYPE_UNSPECIFIED (0): Unset value, which will default to one of the other enum values.
+- RETRIEVAL_QUERY (1): Specifies the given text is a query in a search/retrieval setting.
+- RETRIEVAL_DOCUMENT (2): Specifies the given text is a document from the corpus being searched.
+- SEMANTIC_SIMILARITY (3): Specifies the given text will be used for STS.
+- CLASSIFICATION (4): Specifies that the given text will be classified.
+- CLUSTERING (5): Specifies that the embeddings will be used for clustering.
+
```

Comment:
I suggest we lowercase the values as used with parameter "task_type" as used below.